packages/runtime/src/types.ts

```diff
@@ -215,10 +221,11 @@ export interface AccountStoreI {
   balance: () => bigint
   getApp: (appID: number) => SSCAttributesM | undefined
   getAppFromLocal: (appID: number) => AppLocalStateM | undefined
-  addApp: (appID: number, params: AppDeploymentFlags,
```

Comment:
why are we removing `addApp` from definition here?

Reply:
Uhm, I remember i added it again. I will fix it.